src/components/ListItem.tsx

```diff
@@ -1,18 +1,98 @@
+import MoreHorizIcon from "@mui/icons-material/MoreHoriz";
+import { Button, IconButton } from "@mui/material";
+import AddIcon from "@mui/icons-material/Add";
+import AddCardIcon from "@mui/icons-material/AddCard";
+import { useState } from "react";
+import CloseIcon from "@mui/icons-material/Close";
+import FormControlUnstyled from "@mui/base/FormControlUnstyled";
+import { TextField } from "@mui/material";
+import CardItem from "./carditem";
 
-interface  props {
-    title: string,
-}
+// interface props {
+//   title: string;
+// }
 
-const CardProps = {
-    title: 'rfergv',
+interface cardbox {
+  listcard: {list: string}[];
 }
 
-export default function ListItem(props: props) {
-    return (
-        <div
-         >
-            <div>{CardProps.title}</div>
-            <div>ewfderf</div>
-         </div>
-    );
-}
\ No newline at end of file
+
+export default function ListItem(props: { title: string }) {
+  const [handlecard, setHandleCard] = useState(false);
+  const cardHandler = () => {
+    setHandleCard(true);
+  }
+  const [cardlist, setCardList] = useState("");
```

Comment:
this variable is used for card title right ? please give a proper name

Reply:
resolved